### ISSUE TITLE
Creating Angular Elements project TypeError: paramCase is not a function

### ISSUE THREAD

**Issue by gonadn** [pnp/generator-spfx#222]
#### General Information
* [ ] Usage
* [* ] Development
* [ ] Documentation
* [ ] Feature Request

#### What Version you are running?
* NodeJS: v12.14.1
* NPM: 6.13.4
* @pnp/spfx: latest

Getting error when i try to create Angular Elements project.
Here is all steps in pics:

![image](https://user-images.githubusercontent.com/3224441/72142391-d92cf500-3394-11ea-96f6-91cd43f4454b.png)

![image](https://user-images.githubusercontent.com/3224441/72142444-fd88d180-3394-11ea-862f-f55c368ddf30.png)

![image](https://user-images.githubusercontent.com/3224441/72142468-0c6f8400-3395-11ea-8fea-7f77acdefbbf.png)

![image](https://user-images.githubusercontent.com/3224441/72142489-172a1900-3395-11ea-82d8-df08b1221fce.png)

![image](https://user-images.githubusercontent.com/3224441/72142531-2ad57f80-3395-11ea-8fd5-e0acd6e65160.png)

![image](https://user-images.githubusercontent.com/3224441/72142557-3a54c880-3395-11ea-80aa-4be3add57205.png)

![image](https://user-images.githubusercontent.com/3224441/72142585-4e002f00-3395-11ea-809e-dadd399d45b3.png)

![image](https://user-images.githubusercontent.com/3224441/72142615-5d7f7800-3395-11ea-9bc2-354bfc6620d3.png)

![image](https://user-images.githubusercontent.com/3224441/72142927-0e861280-3396-11ea-80ad-e0e0c9018d64.png)

**Comment by StfBauer** [comment]
Thank you very much @gonadn - The version you have installed supposed to be just an early alpha for the next release. Do you see the same behaviour with 1.10.2?

**Comment by fussionlab** [comment]
@StfBauer  

> Thank you very much @gonadn - The version you have installed supposed to be just an early alpha for the next release. Do you see the same behaviour with 1.10.2?
![image](https://user-images.githubusercontent.com/23405898/72223222-82364400-357d-11ea-89aa-2919467416a8.png)

![image](https://user-images.githubusercontent.com/23405898/72223213-75195500-357d-11ea-967f-4ae8f1f970e6.png)
Version 1.10.2 throws error. TypeError: paramCase is not a function

**Comment by vinaunder** [comment]
I have the same error.
in my case on folder node_modules "param-case" exists.
Check in your project if this folder exists.

**Comment by StfBauer** [comment]
@vinaunder just out of curiosity. A friend of mine was able to reproduce it but only when she cancelled the npm installation process. You haven't pressed Ctrl+C to cancel npm install?

**Comment by vinaunder** [comment]
> @vinaunder just out of curiosity. A friend of mine was able to reproduce it but only when she cancelled the npm installation process. You haven't pressed Ctrl+C to cancel npm install?


I will try to do the whole installation again because that day I found that the company Nexus was having trouble downloading the npm packages. so this may be what happened

**Comment by fussionlab** [comment]
@StfBauer @vinaunder 
I just get through this by removing
```const paramCase = require('param-case') ```
from 11 line `npm\node_modules\@pnp\generator-spfx\generators\angularelements\index.js` and then added the function
```
var tslib_1 = require("tslib");
var dot_case_1 = require("dot-case");

function paramCase(input, options) {
    if (options === void 0) { options = {}; }
    return dot_case_1.dotCase(input, tslib_1.__assign({ delimiter: "-" }, options));
}

```
Now it gets through and on a long drive.

**Comment by fussionlab** [comment]
@StfBauer @vinaunder There is a small TypeError inline 127 and 129 in file
```C:\Users\........\AppData\Roaming\npm\node_modules\@pnp\generator-spfx\generators\angularelements\index.js:```
In line 11 
```const paramCase = require('param-case')```` defined here it's just a `constant varaible` and this varaiable can't act as a function so we can change some thing like this

``` const p = require('param-case') ```` in line 11.
then in line 127 and 129 call that variable
```
   const ejsInject = {
            angularSolutionName: angularSolutionName,
            angularSolutionNameKebabCase: p.paramCase(angularSolutionName),
            componentClassName: manifest.componentClassName,
            componentClassNameKebabCase:p.paramCase(manifest.componentClassName)
        };
```
So that it work

**Comment by StfBauer** [comment]
Thank you @fussionlab for researching this issue. Will be added to the next version.

**Comment by StfBauer** [comment]
@fussionlab do you have your sample project still somewhere. Just out of curiosity can you try the following code instead?

```javascript
import { paramCase } from 'param-case'
```

Potentially you should not need to define a p variable.

**Comment by fussionlab** [comment]
> ```js
> import { paramCase } from 'param-case'
> ```
If you use 
```                 import { paramCase } from 'param-case'         ```
it throw a error that :
```SyntaxError: Unexpected token {```
![image](https://user-images.githubusercontent.com/23405898/72445793-599f8b80-37c3-11ea-9c75-7b5188762f9b.png)

We can do one thing 
``` 
     const p = require('param-case');
     const paramCase = p.paramCase;
````
then it work as normal.
![image](https://user-images.githubusercontent.com/23405898/72445903-8c498400-37c3-11ea-9ff6-7f2e9dd43302.png)

**Comment by StfBauer** [comment]
@fussionlab which NodeJS version are you using?

**Comment by fussionlab** [comment]
> @fussionlab which NodeJS version are you using?

``` 10.18```

**Comment by StfBauer** [comment]
That should work @gonadn used 12 which is still unsupported.

**Comment by StfBauer** [comment]
is fixed in version 1.11.0: Can someone of you confirm it please?
